tests/schemas/cy/test_language.json

```diff
@@ -17,8 +17,11 @@
             "type": "string"
         }
     ],
+    "hub": {
+        "enabled": true
+    },
     "submission": {
-        "button": "Testun cyflwyno",
+        "button": "Cnaipe aighneachta",
```

Comment:
Not sure what happened here, but this looks like Gaelic rather than Welsh 😕 .

Reply:
They are both Welsh - Testun cyflwyno = Presentation text, Cnaipe aighneachta = Submission button.